molecule/yaml_config_validaton/converge.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
There is a typo in the scenario name => s/validaton/validation (you need to rename the folder)